data/sites.json

```diff
@@ -67,5 +67,11 @@
         "zip": "01550",
         "website": "https://signup.com/api/events?accesskey=6775657374&activity_id=3500438&enddate=2021%2F12%2F31&include_comments=false&include_jobassignments=true&include_jobs=true&my_jobs=false&selected_activity=3500438&startdate=2020%2F12%2F16",
         "signUpLink": "https://forms.gle/tMDnRLcXzZnPUxqP9"
+    },
+    "LynnTech": {
+        "name": "Lynn Tech High School Field House",
+        "street": "80 Neptune Blvd",
+        "city": "Lynn",
+        "zip": "01902"
```

Comment:
we're missing a signUpLink (https://www.lchcnet.org/covid-19-vaccine-scheduling#vaccine-screening should do)